src/rdiff_backup/log.py

```diff
@@ -261,7 +268,7 @@ def open(cls, time_string, compress=1):
         assert not cls._log_fileobj, "log already open"
         assert Globals.isbackup_writer
 
-        base_rp = Globals.rbdir.append("error_log.%s.data" % (time_string, ))
+        base_rp = Globals.rbdir.append("error_log.%s.data" % (time_string,))
```

Comment:
In Python 3, the one element list isn't required anymore, I remove them when I stumble upon them.

Reply:
Fixed